deploy/aws/README.md

```diff
@@ -20,26 +20,29 @@ The default setup will create a new VPC and a t2.micro instance as bastion machi
 * 2 c4.4xlarge instances for TiDB
 * 1 c5.xlarge instance for monitor
 
+You can change default values in `variables.tf` (like cluster name and versions) as needed. The default `cluster_name` is `my-cluster`.
 
 ``` shell
-$ git clone https://github.com/pingcap/tidb-operator
+$ git clone --depth=1 https://github.com/pingcap/tidb-operator
 $ cd tidb-operator/deploy/aws
 $ terraform init
 $ terraform apply
 ```
 
-After `terraform apply` is executed successfully, you can access the `monitor_endpoint` using your web browser.
+It might take 10 minutes or more for the process to finish. After `terraform apply` is executed successfully, some basic information will be printed to the console, you can access the `monitor_endpoint` using your web browser.
```

Comment:
```suggestion
It might take 10 minutes or more for the process to finish. After `terraform apply` is executed successfully, some basic information is printed to the console. You can access the `monitor_endpoint` using your web browser.
```